blivet/udev.py

```diff
@@ -99,7 +101,7 @@ def resolve_devspec(devspec):
                     break
 
     if ret:
-        return device_get_name(ret)
+        return ret.sys_name if sysname else device_get_name(ret)
```

Comment:
I still don't like embedding the if like this. I've gotten used to X or Y syntax, but I think this form makes what is happening less clear.